### DISCUSSION TITLE
An interesting name proposed: riceberg

### DISCUSSION THREAD

**Discussion by Xuanwo** [apache/iceberg-rust#469] [Ideas]
One of my friends @ichn-hu suggested a very interesting name for our project that is easy to remember and pronounce. It also makes the project more distinguishable and outstanding.

This is **riceberg**.

Composed by `rust` and `iceberg`. 

We can call it `rice-berg` or in full version: `rust-iceberg`. Also, it aligns with our existing `pyiceberg`.

---

This isn't a proposal to rename the project. I'm sharing this name with the community to get your feedback.

**Comment by liurenjie1024** [comment]
> This isn't a proposal to rename the project. I'm sharing this name with the community to get your feedback.

Do you mean to rename the name on crate.io? TBH I'm not quite comfortable for this abbreivation `riceberg` because it's difficult to remind people without much background of iceberg and rust, and maybe it's not search engine friendly? `rust-iceberg` seems fine for me.

**Comment by Xuanwo** [comment]
> Do you mean to rename the name on crate.io?

No, I don't mean anything. Just to share this name. I'm fine with current name: `iceberg` on crates.io.

Just feel like this name is so interesting that I want the community to know about it.
